### ISSUE TITLE
Small template-level accessibility issues

### ISSUE THREAD

**Issue by mtthwhggns** [apluslms/a-plus#628]
Two small known accessibility issues:

- Text contrast in the page footer
- The gradient in the top navigation area makes automated contrast checks fail (and it looks a little out of place)

Also #609 came up here, but already had an issue.

**Comment by jaguarfi** [comment]
@Mankro Do you know what exactly is "The gradient in the top navigation area"?

**Comment by mtthwhggns** [comment]
> @Mankro Do you know what exactly is "The gradient in the top navigation area"?

I can help with this.

The `.navbar` has the background:

```
background-image: linear-gradient(to bottom, #337ab7, #2e6da4);
```

A blue-blue gradient which creates a shiny effect. A solid colour would allow for automated contrast checks, and would match the rest of the interface's "flat" style.

**Comment by jaguarfi** [comment]
@mtthwhggns again thank you so much for taking the time to clarify this issue. 👍🏽

**Comment by Mankro** [comment]
In addition, the shade of blue in the top navbar was not updated in the summer 2020 when it was changed in the boxes and borders elsewhere in the course frontpage. Therefore, the top navbar now looks very out of place. We can try using the same darker blue in the top navbar as elsewhere (and with a solid colour without the colour gradient). Let's see how that looks.
